### DISCUSSION TITLE
Selected node style question

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#718] [Help] [answered]
When I select a node, FP changes whatever the note looked like to its own style. So if a node was set to Bold, selecting that node removes the Bold attribute.

I would like it if the Selected node style changed _only_ the node's background color, leaving all other stylistic text attributes alone.

Is there a way to do this?

~ Bal

**Comment by macmarrum** [comment, marked as answer]
My **Selected node** style has all formatting properties unset (no revert button next to them) except for background color. This results in what you described: _the Selected node style changed only the node's background color, leaving all other stylistic text attributes alone_

Alternatively I can use `Preferences…->Defaults->Selection colors->Display selected nodes in bubbles`.

**Reply by wordmuse** [reply]
It was the "revert" buttons in my instance of Freeplane. Have they always been there? This is the first I noticed them or understood what they were.

In any event, thank you!  That did the trick. :)

Regards,
~ Bal
